addons/im_livechat/migrations/10.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,14 @@
+---Fields in module 'im_livechat'---
+---XML records in module 'im_livechat'---
+NEW ir.actions.act_window: im_livechat.im_livechat_canned_response_action
+NEW ir.actions.act_window.view: im_livechat.mail_channel_action_livechat_form
+NEW ir.actions.act_window.view: im_livechat.mail_channel_action_livechat_tree
+DEL ir.actions.client: im_livechat.ir_action_client_open_livechat_menu
```

Comment:
I used the `compare_noupdate_xml_records` script and the result was nothing. The `ir.actions.*`.  always are ignored. If you insist, please, show me an existing example of removing `ir.actions.*`